### PR TITLE
Add extra Script for WSL Ubuntu to More Easily build

### PR DESCRIPTION
**Pull request by FREEWING-JP**
Add extra Script for WSL Ubuntu to More Easily build .

Currently requires some patches to build, and the cross_compile_ffmpeg.sh script also needs to be run a few times.
These steps can be difficult for beginners.
I added a fully automated patch and build script for WSL Ubuntu.
By using this script, you can build the FFmeg binary by executing the script only once.

### CODE REVIEW

**Comment by Tupsi** [comment]
That looks like just what I need. Just installed the helper script and having a hard time getting it going because of missing tesseract stuff. Seeing in your files that talk about disabling that it might help me, because otherwise this is just not working in wsl2 ubuntu.

**Comment by FREEWING-JP** [comment]
Thanks !

**Comment by rdp** [comment]
Thanks! Do you remember what the binfmt thing was for?  I can't remember at this point LOL.  I might move these patches inline someday, cheers!

File: extra/wsl_patch.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+# ERROR: tesseract not found using pkg-config
+# Disable Tesseract OCR function
+sed -i -e "s/ build_libtesseract/ # build_libtesseract/g" cross_compile_ffmpeg.sh
+sed -i -e "s/--enable-libtesseract//g" cross_compile_ffmpeg.sh
+
+echo "Windows WSL 'binfmt'"
+if [ -f /proc/sys/fs/binfmt_misc/WSLInterop ]; then
+  echo "Windows WSL Disable 'binfmt'"
+  sudo bash -c 'echo 0 > /proc/sys/fs/binfmt_misc/WSLInterop'
+fi
```
Comment (FREEWING-JP) [inline extra/wsl_patch.sh:18]: binfmt

**Comment by FREEWING-JP** [comment]
> Thanks! Do you remember what the binfmt thing was for? I can't remember at this point LOL. I might move these patches inline someday, cheers!

I forget this too .
Why disable binfmt ?

**Comment by rdp** [comment]
I think it was something like fonttest.exe getting run...I'll test it out and see if I can repro it... thanks!

**Pull request merged**